### PR TITLE
color and background changed

### PR DESCRIPTION
**Pull request by deveshd2k**

### CODE REVIEW

File: templates/home/home.html [hunk 1]
```diff
@@ -2,10 +2,11 @@
 {% load staticfiles %}
 {% block title %}Programming Club {% endblock %}
 {% block content %}
-<div class="jumbotron">
+<div class="jumbotron" style="background-image: url(/static/home/images/image_for_back.jpg); background-size: 100% 100%;" >
```
Comment (divyam3897) [inline templates/home/home.html]: I would change the url to url({% static 'home/images/image_for_back.jpg' %}) , it's not a good practice to hardcode things.

Comment (deveshd2k) [reply templates/home/home.html]: should we cover about us also with image in order to increase the height?

Comment (divyam3897) [reply templates/home/home.html]: No, just increase the complete jumbotron section height

**Comment by gurpreetsingh00885** [comment]
You need to do a rebase. The conflicts will go away. 
And also the code style is not good in base.css.

**Comment by divyam3897** [comment]
Duplicate of https://github.com/pclubuiet/website/pull/104 so closing this one. Please don't resend PRs for same issues repeatedly like this and try to fix your issues.

**Pull request closed without merging**